src/components/calcite-tree/calcite-tree.tsx

```diff
@@ -197,5 +197,8 @@ export class CalciteTree {
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * @todo doc
```

Comment:
Emitted when user selects/deselects tree items. An array of selected items will be passed in the event's `detail`